src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -212,6 +230,9 @@ int WriteLog<I>::append_op_log_entries(GenericLogOperations &ops)
 
   /* Drain once for all */
   pmem_drain();
+  if (m_replica_pool) {
+    m_replica_pool->flush();
+  }
```

Comment:
it can directly call drain_local_and_replica().

Reply:
Replaced.